workers/unity/ProjectSettings/EditorSettings.asset

```diff
@@ -14,8 +14,9 @@ EditorSettings:
   m_EtcTextureFastCompressor: 1
   m_EtcTextureNormalCompressor: 2
   m_EtcTextureBestCompressor: 4
-  m_ProjectGenerationIncludedExtensions: txt;xml;fnt;cd
+  m_ProjectGenerationIncludedExtensions: txt;xml;fnt;cd;asmdef
```

Comment:
do we need that change?

Reply:
I thought we should have committed this in already- I remember overhearing people talking about this file. I believe it comes from when we last upgraded Unity version, whenever you save the project it'd appear again.